app/uploaders/attachment_uploader.rb

```diff
@@ -1,6 +1,6 @@
 class AttachmentUploader < CarrierWave::Uploader::Base
   storage :fog
-
+  # storage :file
```

Comment:
Actually, it may not be necessary to specify storage if it's defined in the configuration file. Can you check if that's the case?